eks/core.py

```diff
@@ -130,31 +136,28 @@ def params_nlgssm_for_keypoint(m0, S0, Q, s, R, f_fn, h_fn) -> ParamsNLGSSM:
 # ----------------- Public API -----------------
 @typechecked
 def run_kalman_smoother(
-    ys: jnp.ndarray,                 # (K, T, obs)
-    m0s: jnp.ndarray,                # (K, D)
-    S0s: jnp.ndarray,                # (K, D, D)
-    As: jnp.ndarray,                 # (K, D, D)
-    Cs: jnp.ndarray,                 # (K, obs, D)
-    Qs: jnp.ndarray,                 # (K, D, D)
-    ensemble_vars: np.ndarray,       # (T, K, obs)
+    ys: jnp.ndarray,                  # (K, T, obs)
+    m0s: jnp.ndarray,                 # (K, D)
+    S0s: jnp.ndarray,                 # (K, D, D)
+    As: jnp.ndarray,                  # (K, D, D)
+    Cs: jnp.ndarray,                  # (K, obs, D)
+    Qs: jnp.ndarray,                  # (K, D, D)
+    ensemble_vars: np.ndarray,        # (T, K, obs)
     s_frames: Optional[List] = None,
     smooth_param: Optional[Union[float, List[float]]] = None,
     blocks: Optional[List[List[int]]] = None,
     verbose: bool = False,
     # JIT-closed constants:
     lr: float = 0.25,
     s_bounds_log: tuple = (-8.0, 8.0),
-    tol: float = 1e-3,
-    safety_cap: int = 5000,
+    tol: float = 1e-2,
+    safety_cap: int = 300,
```

Comment:
just to verify, these changes do not lead to appreciable differences for the linear model predictions right?